src/sentry/api/endpoints/group_notes_details.py

```diff
@@ -45,9 +45,16 @@ def put(self, request, group, note_id):
         serializer = NoteSerializer(data=request.data)
 
         if serializer.is_valid():
+            payload = serializer.validated_data
+            # TODO adding mentions to a note doesn't do subscriptions
```

Comment:
Probably makes sense. We could unsubscribe anyone who was subscribed for the mention reason and were removed from the comment as well.

Reply:
Removing folks is hard, as they could be removed from one comment but still present on another. Adding with existence checks should be safe though.